packages/velo-external-db-core/test/drivers/data_provider_test_support.js

```diff
@@ -12,7 +12,7 @@ const dataProvider = {
 }
 
 const givenListResult = (entities, forCollectionName, filter, sort, skip, andLimit) =>
-    when(dataProvider.find).calledWith(forCollectionName, filter, sort, skip, andLimit)
+    when(dataProvider.find).calledWith(forCollectionName, expect.anything(), sort, skip, andLimit)
```

Comment:
Here we stub the data-provider find function that `data.js` uses, because of the new filter I need the filter that the `filterTransformer.transform` creates and I don't have it, because of this I changed it to `expect.anything()` 
@noam-almog  Is it terrible?

Reply:
i don't understand why can't you know what the input is